ansible/pvp_dut_new.yml

```diff
@@ -0,0 +1,128 @@
+- name: pvp ovs-kernel DUT setup
+  hosts: usa
+  vars_files:
+    - ./trex_settings.yml
+  tasks:
+  - debug:
+      msg: Debug mode is enabled
+    when: redhat_debug_mode == true
+
+
+#############################
+# Return back isolated CPUs #
+# ###########################
+
+  - name: Checking isolcpus in tuned
+    shell: "cat /etc/tuned/cpu-partitioning-variables.conf | grep isolated_cores={{ vcpu_1 }},{{ vcpu_2 }},{{ vcpu_3 }},{{ vcpu_4 }}"
+    register: tuned_found
+    ignore_errors: True
+  - name: Change isolcpus in tuned
+    when: tuned_found.rc != 0
+    replace:
+      dest: /etc/tuned/cpu-partitioning-variables.conf
+      regexp: 'isolated_cores={{ dut_isolated_cpus }}'
+      replace: 'isolated_cores={{ vcpu_1 }},{{ vcpu_2 }},{{ vcpu_3 }},{{ vcpu_4 }}'
+
+  - name: Starting cpu-partitioning tuned profile
+    shell: tuned-adm profile cpu-partitioning
+
+  - name: Checking isolcpus in grub
+    shell: "cat /etc/default/grub | grep isolcpus={{ vcpu_1 }},{{ vcpu_2 }},{{ vcpu_3 }},{{ vcpu_4 }}"
+    register: grub_found
+    ignore_errors: True
+  - name: Change isolcpus in grub
+    when: grub_found.rc != 0
+    replace:
+      dest: /etc/default/grub
+      regexp: 'isolcpus={{ dut_isolated_cpus }}'
+      replace: 'isolcpus={{ vcpu_1 }},{{ vcpu_2 }},{{ vcpu_3 }},{{ vcpu_4 }}'
+
+  - name: Tweak the kernel isolated list
+    command: "grub2-mkconfig -o /boot/grub2/grub.cfg"
+    when: grub_found.rc != 0
+
+###################################################
+# Reboot dut system for settings to take effect #
+###################################################
+
+  - name: REBOOTING DUT SERVER NOW
+    reboot:
+    when: grub_found.rc != 0
+
+#######################################
+# Release the DPCK NIC back to kernel #
+# #####################################
+
+
+  - name: Delete ovs bridge
+    shell: ovs-vsctl --if-exists del-br ovs_pvp_br0
+  - name: Disable dpdk
+    shell: ovs-vsctl set Open_vSwitch . other_config:dpdk-init=false
+  - name: Stop openvswitch
+    service:
+      name: openvswitch
+      state: stopped
+  - name: Unbind nic from dpdk
+    shell: driverctl -v unset-override {{ dut_interface_1_pciid }}
+  - name: Start openvswitch
+    service:
+      name: openvswitch
+      state: started
+
+#################
+# Re-config OVS #
+#################
+
+  - name: Delete ovs_pvp_br0 if exist
```

Comment:
Looks like you already do this above.  Is it needed again here?

Reply:
yes, really a duplication, jut to confirm twice. How about changing it and create another pull request?